shortly/app/entrypoints/http/fastapi/main.py

```diff
@@ -0,0 +1,9 @@
+from fastapi import FastAPI
+
+from shortly.app.entrypoints.http.fastapi.middlewares import setup_middlewares
+from shortly.app.entrypoints.http.fastapi.signals import setup_signals
```

Comment:
I feel like it would be better to use relative import here instead of absolute. This could be almost like a package.
```python
from .signals import setup_signals
from .middlewares import setup_middlewares
```